released/packages/coq-category-theory/coq-category-theory.1.0.0/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "johnw@newartisans.com"
+
+homepage: "https://github.com/jwiegley/category-theory"
+dev-repo: "git+https://github.com/jwiegley/category-theory.git"
+bug-reports: "https://github.com/jwiegley/category-theory/issues"
+license: "BSD-3-Clause"
+
+synopsis: "An axiom-free formalization of category theory in Coq"
+description: """
+An axiom-free formalization of category theory in Coq for personal study and
+practical work.
+"""
+
+build: [make "JOBS=%{jobs}%" ]
```

Comment:
Since this project takes a long time to compile, I think we want the standard parallelism boilerplate here (I can't get parallel jobs to work without it locally):
```suggestion
build: [make "-j%{jobs}%"]
```